compiler/x/codegen/ControlFlowEvaluator.cpp

```diff
@@ -1149,11 +1128,7 @@ void OMR::X86::TreeEvaluator::compareBytesForOrder(TR::Node *node, TR::CodeGener
 
 TR::Register *OMR::X86::TreeEvaluator::gotoEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   TR::Compilation *comp = cg->comp();
-   bool needVMThreadDep =
-      comp->getOption(TR_DisableLateEdgeSplitting) ||
-      !performTransformation(comp, "O^O LATE EDGE SPLITTING: Omit ebp dependency for %s node %s\n", node->getOpCode().getName(), cg->getDebug()->getName(node));
-   generateJumpInstruction(JMP4, node, cg, needVMThreadDep);
+   generateJumpInstruction(JMP4, node, cg, false);
```

Comment:
Will the parameters of APIs be cleaned up in a later commit as part of #2256?

Reply:
Yes, that is my plan as part of my incremental approach to untangling this.  Perhaps I'm being overly-conservative, though, so I could be talked into changing it for this PR.

Reply:
> Yes, that is my plan as part of my incremental approach to untangling this. Perhaps I'm being overly-conservative, though, so I could be talked into changing it for this PR.

No need. Just confirming it will happen in the future. Incremental deprecation is appreciated.